docs/disconnectedCallback.md

```diff
@@ -0,0 +1,28 @@
+@typedef {function} can-component/disconnectedCallback disconnectedCallback
+@parent can-component.events
+
+@description An event called with the component’s element during teardown after bindings are removed.
+
+@signature `connectedCallback: () => function disconnectedCallback (element) { ... }`
+
+Used to teardown anything that was set up during the [can-component/connectedCallback] event. `disconnectedCallback` is defined by returning a function from the [can-component/connectedCallback] event. (this allows it to be defined within the same closure scope as setup)
```

Comment:
“teardown” -> “tear down” because it’s used as a verb

Also, I’d take the sentence out of the parenthesis.